design/ceph/node-loss-rbd-rwo.md

```diff
@@ -0,0 +1,68 @@
+# Recover RBD RWO Pod in case of Node Loss
+
+## Goal
+
+Faster RBD RWO recovery in case of node loss.
+
+## Problem
+
+When a  node is lost where a pod is running with the RBD RWO volume is mounted. The volume cannot be mounted on another node since it could cause corruption if two clients are writing to the same volume.
+
+## Current Solution
+
+We have a manual solution to the problem which involves forceful deletion of a pod so that forced detachment and attachment work is possible. The problem with the current solution is that it takes around 11min to recover and also there are still chances of data corruption when the old pod on the lost node comes back online, this can cause multiple writers and lead to data corruption.
```

Comment:
```suggestion
We have a manual solution to the problem which involves forceful deletion of a pod so that forced detachment and attachment work is possible. The problem with the current solution is that even after the forced pod deletion it takes around 11 minutes for the volume to mount on the new node. Also there are still chances of data corruption if the old pod on the lost node comes back online, causing multiple writers and lead to data corruption.
```